courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/distributed_training.ipynb

```diff
@@ -58,6 +58,7 @@
         "colab": {}
```

Comment:
<p>Line #1, change python to Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>